assets/js/handlebars/templates/versions-dropdown.handlebars

```diff
@@ -2,7 +2,10 @@
   <label>
     <span class="sidebar-projectVersionsDropdownCaret" aria-hidden="true">&#x25bc;</span>
     <span class="sr-only">Project version</span>
-    <select class="sidebar-projectVersionsDropdown">
+    {{#if latestVersion}}
+      <i class="ri-alert-line sidebar-staleIcon" aria-hidden="true"></i>
+    {{/if}}
+    <select class="sidebar-projectVersionsDropdown{{#if latestVersion}} sidebar-staleVersion{{/if}}"{{#if latestVersion}} title="This version is not the latest version of the package."{{/if}}>
```

Comment:
I think that this should be both more SR friendly and mobile friendly with the use of a [popover](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API/Using). I am thinking:

```suggestion
{{#if latestVersion}}
  <i class="ri-alert-line sidebar-staleIcon" aria-hidden=true popovertarget="staleVersion"></i>
  <span class="sr-only">This is not the latest version of this package.</span>
  <div id="staleVersion" popover>This is not the latest version of this package.</div>
{{/if}}
<select class="sidebar-projectVersionsDropdown{{# if latestVersion}} sidebar-staleVersion{{/if}}">
```

I’m not sure if this will work, though.

Reply:
Since popover works with clicks and not hover, this won't work as clicking on the icon will open the version selector.

Reply:
SR text addressed in da05c1d9